content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -165,10 +165,10 @@ your cluster. Those fields are:
   {{< note >}}
   <!--
   The `minDomains` field is a beta field and enabled by default in 1.25. You can disable it by disabling the
-  `MinDomainsInPodToplogySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+  `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
   -->
   `minDomains` 字段是一个 Alpha 字段，在 1.25 中默认被启用。
-  你可以通过禁用 `MinDomainsInPodToplogySpread`
+  你可以通过禁用 `MinDomainsInPodTopologySpread`
   [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)来禁用该字段。
```

Comment:
upstream main branch have updated the content. Please resync the file.
```
./scripts/lsync.sh content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.m
```

see https://github.com/kubernetes/website/pull/38566